nixos/modules/services/games/asphyxia-core.nix

```
@@ -0,0 +1,158 @@
+{ config, lib, options, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.asphyxia-core;
+  opt = options.services.asphyxia-core;
+  configAsIni = generators.toINIWithGlobalSection {} {
+    globalSection = {
+          port = cfg.port;
+          bind = cfg.bind;
+          ping_ip = cfg.pingIp;
+          matching_port = cfg.matchingPort;
+          allow_register = cfg.allowRegister;
+          maintenance_mode = cfg.maintenanceMode;
+          enable_paseli = cfg.enablePaseli;
+    };
```

```suggestion
    globalSection = {
      port = cfg.port;
      bind = cfg.bind;
      ping_ip = cfg.pingIp;
      matching_port = cfg.matchingPort;
      allow_register = cfg.allowRegister;
      maintenance_mode = cfg.maintenanceMode;
      enable_paseli = cfg.enablePaseli;
    };
```
This should be RFC42 style and then we don't need to map any options.